### PR TITLE
Update cowsay.md

### PR DESCRIPTION
**Pull request by jeeftor**
Print for the different options

### CODE REVIEW

File: pages/common/cowsay.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # cowsay
 
 > Generate an ASCII character like a cow or sheep saying or thinking something.
+> Available characters are stored in the /usr/share/cowsay on Linux.
+> And /usr/local/share/cows/ on Mac. directory.
```
Comment (agnivade) [inline pages/common/cowsay.md]: Please remove the "directory" after Mac.

**Comment by jeeftor** [comment]
Any update on merging this?

**Comment by agnivade** [comment]
@waldyrious is yet to give his :+1:

**Comment by waldyrious** [comment]
Sorry guys 😞 I'll propose a separate PR for discussing feedback I'd have added here.

**Pull request merged**